code-studio/package.json

```diff
@@ -10,21 +10,29 @@
   },
   "scripts": {
     "postinstall": "linklocal",
-    "build": "browserify --debug src/js/blockly_editor.js -o build/js/blockly_editor.js",
-    "build:min": "browserify src/js/blockly_editor.js | uglifyjs -o build/js/blockly_editor.min.js",
-    "prebuild": "mkdir -p build/js",
-    "prebuild:min": "mkdir -p build/js",
     "clean": "rm -rf build",
-    "env": "env",
-    "lint": "npm run lint:js -s",
-    "lint:js": "jshint src/js/*.js",
+    "lint": "npm run lint-js -s && npm run lint-css -s",
+    "lint-css": "sass-lint src/css/*css",
+    "lint-js": "jshint src/js/*.js",
+    "build": "npm run build-js && npm run build-css",
+    "build:min": "npm run build-js:min && npm run build-css:min",
+    "build-css": "node-sass --include-path node_modules src/css/blockly_editor.css.scss build/css/blockly_editor.css",
+    "prebuild-css": "mkdir -p build/css",
+    "build-css:min": "node-sass --output-style compressed --include-path node_modules src/css/blockly_editor.css.scss build/css/blockly_editor.min.css",
+    "prebuild-css:min": "mkdir -p build/css",
+    "build-js": "browserify --debug src/js/blockly_editor.js -o build/js/blockly_editor.js",
+    "prebuild-js": "mkdir -p build/js",
+    "build-js:min": "browserify src/js/blockly_editor.js | uglifyjs -o build/js/blockly_editor.min.js",
+    "prebuild-js:min": "mkdir -p build/js",
```

Comment:
Yes: The prebuild step has been broken out into `prebuild-js` and `prebuild-css` and since `build` now simply runs `build-js` and `build-css` both of those prebuild steps get implicitly invoked.